### DISCUSSION TITLE
Testing that everything works!

### DISCUSSION THREAD

**Discussion by RespectableGlioma** [nyumets/nyumets.github.io#2] [General]
@katielink @zane292 Everything should be up and running in a week!

**Comment by katielink** [comment]
Things that I still need to do for the website:
- [ ] Add imaging data metadata table
- [ ] Add dataset page with some of the figures from the paper?
- [ ] Make small sample dataset (~10 patients) instantly available to anyone
- [ ] Write short example sagemaker notebook loading the data
- [ ] Write API section for some of the NYUMets code if used in the sagemaker notebook

Let me know if I missed anything! Obviously science is the highest priority, but these should still be done before the blog post.

**Reply by katielink** [reply]
Another nice-to-have will be a FAQ section, especially for data access issues.

**Reply by RespectableGlioma** [reply]
Agreed on all points above, especially a FAQ. Maybe do first thing once the paper is submitted? Also could be good for one of the students to work on. Perhaps a good place for Karl Lee to start?
